on_device_training/web/web-bundler/src/mnist.ts

```diff
@@ -0,0 +1,119 @@
+// Modified from https://github.com/cazala/mnist/blob/master/src/mnist.js
+// so that we can place the data in a specific folder and avoid out of memory errors
+// and use TypeScript.
+// TODO: modify comments to credit justin
```

Comment:
Do you intend to tackle this TODO in this PR or in a subsequent PR?

Reply:
oops -- updated in last commit!! sorry I thought I had addressed all the TODO comments